staging/src/k8s.io/apiserver/pkg/server/options/serving_with_loopback.go

```diff
@@ -79,3 +99,32 @@ func (s *SecureServingOptionsWithLoopback) ApplyTo(secureServingInfo **server.Se
 
 	return nil
 }
+
+func (s *SecureServingOptionsWithLoopback) NewLoopbackClientCert() ([]byte, []byte, error) {
```

Comment:
I wonder if it would be better to have this just set s.LoopbackCertPEM and s.LoopbackKeyPEM instead of returning the []bytes to the caller (and then making them set those 2 fields separately)?

Reply:
prefer the current factoring. In the new way proposed here it would be more like an on-demand defaulter. That's odd.

Haven't checked whether `NewLoopbackClientCert` uses anything from  `s`. I guess it does.

Reply:
Ok, I can keep it like this.